### DISCUSSION TITLE
``forge create SimpleStorage``  giving error

### DISCUSSION THREAD

**Discussion by vishnuJANGID4564** [Cyfrin/foundry-full-course-cu#1840] [General]
when i am typing the command `` $ forge create SimpleStorage --interactive``
i am getting an error saying "error sending request to the url "http://127.0.0.1:8545" 
![Screenshot 2024-06-16 162624](https://github.com/Cyfrin/foundry-full-course-f23/assets/147773134/f54e347f-7272-457b-8459-c398aaadd903)
please help me out of this error i have tried chatgpt but I am finding it difficult to solve the issue

**Comment by urosognjenovic** [comment]
Make sure you have Anvil running on your local host.

Open another tab in terminal, type ```anvil```, click Enter and that will spin up Anvil. After that, go back to the first terminal and run ```forge create SimpleStorage --interactive```.

**Reply by vishnuJANGID4564** [reply]
Oh! yeah. That was the issue. 
Thank You very much dude.

**Comment by 0xprivateChaos** [comment]
Hey @vishnuJANGID4564 you are sending request to a url where node is absent, so instead use this command 
```forge create SimpleStorage --rpc-url rpc-url --interactive```
At the place of "rpc-url" paste it from ganache or anvil rpc-url.

**Comment by EngrPips** [comment]
Hello @vishnuJANGID4564, Did you figure this out already?
